helm/kubeblocks/falkordb-cluster-omnistrate/tests/integration/replication/test_replication_integration.py

```
@@ -0,0 +1,1922 @@
+        patch_cmd = [
+            "kubectl", "patch", "cluster", cluster_name,
+            "-n", namespace,
+            "--type", "json",
+            "-p", json.dumps([{
+                "op": "replace",
+                "path": f"/spec/componentSpecs/{component_index}/replicas",
+                "value": current_replicas + 1
+            }])
+        ]
+        
+        subprocess.run(patch_cmd, capture_output=True, text=True, timeout=30)
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)